router/router.go

```diff
@@ -0,0 +1,26 @@
+package router
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/gorilla/mux"
+	"github.com/raymundo/academy-go-q42021/controller"
+	"github.com/raymundo/academy-go-q42021/global"
+)
+
+func NewRouter() *http.Server {
+	router := mux.NewRouter()
+
+	router.HandleFunc("/pokemon/{name}", controller.GetPokemonByName).Methods("GET")
+	router.HandleFunc("/pokemonid/{id}", controller.GetPokemonById).Methods("GET")
```

Comment:
It's better to have a common route for the same paths.

You might struggle with this, because it's the same route; in this scenario, I'd merge both controllers into a single one, and to differentiate them would be using a query param, to denote if the search is going to be by id, by name, or by any other field.

```suggestion
	router.HandleFunc("/pokemons/{field}", controller.GetSinglePokemon).Methods("GET")
```

And also I'd add a controller for the route `/pokemons` to list all the pokemon in our data source.